README.md

```diff
@@ -296,7 +296,20 @@ Other interesting functions you can use are: `randBytes`, `htpasswd`, `encryptAE
 Sometimes you have tests that work well for some platform combinations, e.g. Apache + modsecurity2, but fail for others, e.g. NGiNX + modsecurity3. Taking that into account, you can override test results using the `testoverride` config param. The test will be skipped, and the result forced as configured.
 
 Tests can be altered using four lists:
-- `input` allows you to override global parameters in tests. An example usage is if you want to change the `dest_addr` of all tests to point to an external IP or host
+- `input` allows you to override global parameters in tests. The following ones can be overridden:
+  - `dest_addr`: allows to point to an external IP or host
+  - `override_empty_host_header`: enabling this flag, empty Host headers will be replaced with `dest_addr`
+  - `port`: overrides the port number
+  - `protocol`: overrides the protocol
+  - `uri`: overrides the uri
+  - `version`: overrides the HTTP version. E.g. "HTTP/1.1"
+  - `headers`: overrides headers, the format is a map of strings
+  - `method`: overrides the method used to perform the request
+  - `data`: overrides data sent in the request
+  - `savecookie`: 
+  - `stopmagic`:
```

Comment:
If we keep this option, we should rename it IMO (keep it for backward compatibility but add a sibling that has a better name and document that here). @fzipi?

```suggestion
  - `stopmagic`: prevent header autocompletion (currently sets `Connection: close` and `Content-Length` for requests with body data)
```

Reply:
Yes `stopmagic` is clearly not a good name regarding its semantic. We can use a better one.

Reply:
I have a draft PR waiting to rename stopmagic after this PR has been merged.